main.rb

```diff
@@ -1,8 +1,87 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
 require './app'
 
-def main
-  app = App.new
-  app.run
+class Main
+  include App
+  def initialize
+    @books = []
+    @persons = []
+  end
+
+  def enter_msg
+    print "\n#{ENTER_MSG}"
+    gets
+  end
+
+  def create_person
+    puts "\nPlease choose a option by entring a number
+    1 -  Create a teacher
+    2 -  Create a student"
+    print "\n#{INPUT_MSG}"
+    option = gets.chomp.to_i
+    case option
+    when 1
+      @persons.push(create_teacher)
+      puts 'Teacher created successfully!'
+    when 2
+      @persons.push(create_student)
+      puts 'Student created successfully!'
+    else
+      puts 'Please select a correct option'
+      create_person
+    end
+  end
```

Comment:
- [ ] according to the requirements you should Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit). please place this method somewhere else.
- [ ] In this method you are printing to the console, getting user response and processing user response this is not in line with the single responsibility principle, have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information.